website/content/docs/agent/template-config.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Vault Agent Template Config
+description: |-
+  Vault Agent's Template Config to set Templating Engine behavior
+---
+
+# Vault Agent Template Config
+
+Template Config allows users of Vault Agent to determine the behavior that
+pertains to the templating engine.
```

Comment:
```suggestion
Template Config configures default behavior for [templates](/docs/agent/template).
```

Reply:
Perhaps also worth adding a note and the reverse link from the template page?

Reply:
I referenced "templating engine" on purpose instead of `template` because the params within the `template` stanza defines the rendering behavior for templates themselves so I didn't want to confuse the two. The current wording might not be the best though, so still open to suggestions!

Reply:
I'm reading the suggesting as adding something to `/docs/agent/template` referencing _this_ page. I can imagine being very confused when someone is supposed to "set xyz in the template config", and they're on the template docs page looking and a big section called "Configuration" and not finding it. It is still intrinsically confusing, IMO, but as least some extra guidance might route them to the right page.

Reply:
My reply was to Tom's commit suggestion and not on the reverse link. I thin it makes sense to add something to `/docs/agent/template` that references this page though so I'll get that updated!

Reply:
Ended up backlinking both ways. I realized how confusing it can be otherwise. Thanks for the suggestion Tom and Jim!